components/Smiley.tsx

```diff
@@ -18,13 +20,24 @@ const EMOJI_MAP: EmojiMap = {
   90: '😄'
 }
 
-const Smiley = ({ percentage }: Props) => {
+const Smiley = ({ max = 10, min = 0, percentage }: Props) => {
```

Comment:
Is this min and max correct? I think the value reported to the survey server not 0-10. I'd really like to avoid these fixed numbers if possible.  Could we maybe extract the numbers from the emoji map? We could add a 100 value and then look at `Object.keys` of the emoji map to extract the highest and lowest values